### DISCUSSION TITLE
Missing features

### DISCUSSION THREAD

**Discussion by DudeMcGee** [KDE/ghostwriter#853] [General]
1. There is no commandline option to start Ghostwriter in distraction-free-mode. 
2. Ghostwriter does not remember the configuration of its last session, e. g., if it was in distraction-free-mode, it won't start using this mode. The user has to activate this mode by himself.
3. There is no keyboard shortcut to start distraction-free-mode. One always has to use the mouse to click on the icon to start it.

**Comment by subacoustic** [comment]
The keyboard shortcut to quit Ctrl-Q doesn't work on Xubuntu...  Other keyboard shortcuts seem to work

**Comment by hackbarth** [comment]
There's a 'Save' keyboard shortcur: Ctrl+S, but there isn't a 'Save as' keyboard shortcut, such as Ctrl+Shift+S.

As it is the program requires the user to open the file menu for the first time he saves a file, the inclusion of a 'Save as" keyboard shortcut would save time and would benefit from muscle memory, since Ctrl+Shift+S is the De Facto standard for this command.
